test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/XPlatConfigUnitTests.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.CommandLine.XPlat;
+using Xunit;
+
+namespace NuGet.CommandLine.Xplat.Tests
+{
+    public class XPlatConfigUnitTests
+    {
+        [Theory]
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        public void ConfigPathsCommand_WithNullArguments_ThrowsArgumentNullException(bool argsIsNull, bool loggerIsNull )
```

Comment:
nit: remove extra spaces

```suggestion
        public void ConfigPathsCommand_WithNullArguments_ThrowsArgumentNullException(bool argsIsNull, bool loggerIsNull)
```